e2e/passcode/post-passcode-setup.yaml

```diff
@@ -0,0 +1,71 @@
+appId: com.comapeo
+env:
+  CAT: MAIN
+---
+- tapOn:
+    id: '${CAT}.drawer-icon'
+- tapOn: 'Security'
+- tapOn: 'App Passcode'
+- assertVisible:
+    label: 'From Security screen, [On click] App Passcode button prompts password entry'
+    text: 'Enter Passcode'
+- runFlow: './subflows/check-number-keyboard.yaml'
+- tapOn:
+    id: '${CAT}.header-back-btn'
+- assertVisible:
+    label: 'Back button returns to the Security Screen'
+    id: '${CAT}.security-scrn'
+- tapOn: 'App Passcode'
+- runFlow: './subflows/cancel-to-security.yaml'
+- tapOn: 'App Passcode'
+- inputText: ${output.newpasscode}
+- assertVisible:
+    label: 'Next with incorrect passcode displays error'
+    text: 'Incorrect Passcode'
+- inputText: ${output.passcode}
+- assertVisible:
+    label: 'Next with correct passcode opens app passcode screen'
+    text: 'App Passcode'
+- assertVisible:
+    label: 'Screen displays abbreviated feature description'
+    text: '.*App passcode adds an additional layer of security.*'
+- assertVisible:
+    label: 'Use App Passcode checkbox in checked state'
+    id: 'SETTINGS.passcode-checked'
```

Comment:
guessing the answer is no, but could these tests use the `checked` selector?

https://maestro.mobile.dev/api-reference/selectors

I'm guessing it doesn't because we're not using "native" radio elements in various cases (mostly just conditionally rendering icons, which is unfortunate 😬 )

Reply:
Yes exactly. I wish...